src/index.js

```diff
@@ -36,6 +38,12 @@ const extendCodeBlockWithSyntaxHighlighting = ( settings ) => {
 			language: {
 				type: 'string',
 			},
+			selectedLines: {
+				type: 'array',
+			},
+			selectedLinesRaw: {
+				type: 'string',
+			},
```

Comment:
I don't think there should be two copies of the same data in attributes. I think the array version can be eliminated, in favor of deriving the array from the string when rendering.

Reply:
Renamed `selectedLinesRaw` to `selectedLines` and kept it as a `string`.